Sources/SwifterSwift/UIKit/UIViewControllerExtensions.swift

```diff
@@ -111,6 +111,27 @@ public extension UIViewController {
         removeFromParent()
         view.removeFromSuperview()
     }
+    
+    /// SwifterSwift: Helper method to set tabBarItem when UIViewController is added to a UITabBarViewController with image from SF Symbol
+    ///  - Example: UIViewController.setTabBarImage(SFImageName: "message.fill", title: "Messages")
+    ///
+    ///  - Parameters:
+    ///     - imageName: the SF Symbol name.
+    ///     - title: the tab bar title.
+    ///
+    ///  - Throws: assertionFailure occur when SF Symbol is not found
+    func setTabBarImage(SFImageName: String, title: String) {
```

Comment:
Maybe `systemName`? because that is what apple uses on their API's to refer to SFSymbols. See `UIImage` init for example.
```suggestion
    func setTabBarImage(systemName: String, title: String) {
```